src/ParseQuery.js

```diff
@@ -1482,6 +1512,20 @@ class ParseQuery {
     return this;
   }
 
+  /**
+   * Changes the read preference that the backend will use when performing the query to the database.
+   * @param {String} readPreference The read preference for the main query.
+   * @param {String} includeReadPreference The read preference for the queries to include pointers.
+   * @param {String} subqueryReadPreference The read preference for the sub queries.
+   * @return {Parse.Query} Returns the query, so you can chain this call.
+   */
+  readPreference(readPreference: string, includeReadPreference: string, subqueryReadPreference: string): ParseQuery {
```

Comment:
Are all theses fields required? For typing you can do:

`includeReadPreference?: string, subqueryReadPreference?: string`

Reply:
just fixed